website_seo_redirection/models/website_seo_redirection.py

```diff
@@ -127,3 +159,32 @@ def redirect_auto(self, path=None, code=301, website=None, rerouting=None):
             dict(request.httprequest.args),
             True,
```

Comment:
Is it possible that [you are not using Firefox](https://github.com/odoo/odoo/blob/9.0/openerp/http.py#L156-L164)?